_build/templates/default/sass/_tabs.scss

```diff
@@ -165,7 +165,7 @@
 }
 
 ul.x-tab-strip-top li:first-child {
-  /*margin-left: -1px*/;
+  /*margin-left: -1px*/
```

Comment:
Now that this line has been edited it makes more sense to be completely removed.